.github/workflows/benchmark_large.yml

```diff
@@ -13,6 +13,16 @@ on:
     # Scheduled to run at 09:00 UTC and 21:00 UTC.
     - cron: '0 09,21 * * *'
   workflow_dispatch:
+    inputs:
+      shard-count:
+        description: |
+          A device-name to integer mapping as a comma separated list.
+          Allows to assign a distinct shard count for each target device.
+          The reserved keyword `default` assigns a shard count to all target devices
+          that are not explicitly listed.
+        # Please keep the shard count default value in sync with jobs.build_e2e_test_artifacts.with.shard-count
+        default: a2-highgpu-1g=2,c2-standard-16=2,default=1
```

Comment:
There are two things called "default" here and I'm not totally sure which you're talking about in the comment.